granary/meetup.py

```diff
@@ -108,7 +108,7 @@ def _create(self, obj, preview=False, include_link=source.OMIT_LINK, ignore_form
         create_resp = {
                 'url': '%(event_url)s#rsvp-by-%(url)s' % {
                     'event_url': event_url,
-                    'url': urllib.parse.quote_plus(obj['actor']['url']),
+                    'url': urllib.parse.quote_plus(obj['id']),
```

Comment:
ah, the catch here is that you can't assume any specific mf2 property will be there, including `id`. so `.get()` is really the most important change here, not the property itself, with reasonable fallback. the fallback itself can be more than one property, or just one, as long as it does something reasonable like empty fragment when none of those properties are available.